style.css

```diff
@@ -0,0 +1,662 @@
+}
+
+#inner {
+  display: table;
+  width: 70%;
+  table-layout: fixed;
+}
+
+.about_myself {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 40px;
+  margin-top: 60px;
+  margin-bottom: 20px;
+  font-weight: 700;
+  color: #172b4d;
+  line-height: 52px;
+  position: relative;
+}
+
+.popup {
+  height: 100%;
+  width: 0;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  background-image: url(images/Overlay.svg);
+  overflow-x: hidden;
+  transition: 0.5s;
+  padding-top: 60px;
+  text-align: center;
+}
+
+.popup a:hover {
+  color: #818181;
+}
+
+.popup .closebtn {
+  position: absolute;
+  top: 0;
+  right: 25px;
+  font-size: 36px;
+  margin-left: 50px;
+}
+
+.slidefont {
+  font-size: 30px;
+}
+
+@media screen and (min-width: 768px) {
+  .beginning {
+    padding: 0 50px;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .navigation_option {
+    display: flex;
+    align-self: center;
+    justify-self: center;
+    flex-direction: row;
+  }
+
+  .toolbar {
+    min-height: 916px;
+    background-repeat: no-repeat;
+    display: flex;
+    justify-content: center;
+    align-items: flex;
+    margin-left: 100px;
+  }
+
+  .mid {
+    align-self: center;
+    justify-self: center;
+    max-width: 920px;
+  }
+
+  .start {
+    margin: 25px 100px;
+  }
+
+  .topic {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .topic:nth-child(even) .summary {
+    order: 2;
+  }
+
+  .topic2 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .topic2:nth-child(even) .summary {
+    order: 2;
+  }
+
+  .topic3 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .topic3:nth-child(even) .summary {
+    order: 2;
+  }
+
+  .topic4 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .topic4:nth-child(even) .summary {
+    order: 2;
+  }
+
+  .instruction {
+    margin-left: 15px;
+  }
+
+  .information {
+    grid-template-columns: 1fr 1fr;
+    padding: 25px 50px;
+  }
+
+  .purple_square {
+    margin-top: 60px;
+  }
+
+  .portrait {
+    display: table-cell;
+    vertical-align: top;
+    width: 100%;
+  }
+
+  .obtainer {
+    flex-direction: column;
+    align-items: flex-start;
+    width: 120px;
+    height: 120px;
+  }
+
+  .obtainer img {
+    margin-bottom: 15px;
+  }
+
+  .name {
+    color: #fff;
+    padding: 20px;
+    margin-top: 150px;
+    background-size: contain;
+    border-top-left-radius: 70px;
+    background-repeat: no-repeat;
+    background-position: bottom -1px right -13px;
+    background-image: url("images/contact\ form\ background\ shapes.svg");
+    background-color: #6070ff;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .form {
+    display: flex;
+    background-position: initial;
+    background-size: cover;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+  }
+
+  .details {
+    width: 500px;
+    justify-self: center;
+  }
+
+  .popup {
+    padding-top: 15px;
+  }
+
+  .popup a {
+    font-size: 18px;
+  }
+}

```

Comment:
- [ ] As stated the project requirement and suggested by the previous reviewer You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Kindly Work on styling the page just like the Figma design.

Your Project Look likes this : 
 
![Capture](https://user-images.githubusercontent.com/93598189/203550288-d1017616-9faf-4384-80d1-e5d6b3ca273a.PNG)


The Figma design Look this : 
![Capture](https://user-images.githubusercontent.com/93598189/203550473-63f5e130-4f6c-46df-8a4a-4f268e2b9bbf.PNG)